integration/src/integration/java/pl/allegro/tech/hermes/integration/HermesClientPublishingTest.java

```diff
@@ -47,6 +51,38 @@ public void shouldPublishUsingRestTemplate() {
         runTestSuiteForHermesClient(client);
     }
 
+    @Test
+    public void shouldPublishUsingOkHttpClient() {
+        // given
+        HermesClient client = hermesClient(new OkHttpSender()).withURI(topicURI).build();
+
+        // when & then
+        runTestSuiteForHermesClient(client);
+    }
+
+    @Test
+    public void shouldPublishUsingOkHttpClientUsingSSL() {
```

Comment:
does it mean it's not http2, just ssl? i don't know why are we keeping 2 separate tests to check ssl and http2 (or the naming is bad)